README.md

```diff
@@ -134,37 +134,7 @@ The versioning in Filecoin-FFI currently follows the Lotus versioning.  For exam
 
 #### Release Process
 
-##### Cutting a development or release candidate release
-
-<details>
-  <summary>Steps to cut a development or release candidate release:</summary>
-
-1. Go to [Filecoin-FFI Releases](https://github.com/filecoin-project/filecoin-ffi/releases).
-2. Click the "Draft a new release" button in the right corner.
-3. In the "Choose a tag" dropdown, enter the desired version and click "Create new tag: vX.XX.X-rcX/dev on publish".
-4. Target the master branch.
-5. Set the previous tag to compare against, which should be the last stable release.
-6. Click the "Generate release notes" button.
-6. Check the "Set as a pre-release" checkbox.
-7. Click "Publish release" to create the release.
-
-</details>
-
-##### Cutting a definitive release
-
-<details>
-  <summary>Steps to cut a definitive release:</summary>
-
-1. Go to [Filecoin-FFI Releases](https://github.com/filecoin-project/filecoin-ffi/releases).
-2. Click the "Draft a new release" button in the right corner.
-3. In the "Choose a tag" dropdown, enter the desired version and click "Create new tag: vX.XX.X on publish".
-4. Target the release candidate you want to create a stable.
-5. Set the previous tag to compare against, which should be the last stable release (e.g., non-RC)
-6. click the "Generate release notes" button.
-6. Ensure the "Set as a latest release" checkbox **is checked**.
-7. Click "Publish release" to create the release.
-
-</details>
+Create a new entry in the [Changelog](CHANGELOG.md) for the new release. If you don't provide a changelog entry, the release will be created with auto-generated release notes.
```

Comment:
can you clarify this a bit more please - making a new release is done by making a new entry; does it need to be specifically formatted, how does one specify the version? and the second sentence seems to conflict - if you don't provide an entry you'll get a release anyway? so on each commit? what triggers this new release if you're not touching the changelog?

Reply:
```suggestion
Create a new entry in the [Changelog](CHANGELOG.md) for the new release. If you don't provide any description/body for the new entry, the release will be created with auto-generated release notes.
```

Reply:
> does it need to be specifically formatted

Yes, it follows keep a changelog format. I updated the changelog to follow that formatting.

> how does one specify the version

Create a new entry in https://github.com/filecoin-project/filecoin-ffi/blob/3a3603febb8e0d35881be4053fb9d9b569c07517/CHANGELOG.md

> what triggers this new release if you're not touching the changelog?

You have to touch the changelog, I meant entry body. Updated now.

Reply:
ah, that makes sense